lib/modules/modules.go

```diff
@@ -118,7 +121,7 @@ func (f Features) ToProto() *proto.Features {
 		OIDC:                    f.OIDC,
 		SAML:                    f.SAML,
 		AccessControls:          f.AccessControls,
-		AdvancedAccessWorkflows: f.AdvancedAccessWorkflows,
+		AdvancedAccessWorkflows: f.AccessRequests.Enabled,
```

Comment:
Okay seems like this line will solve the backward compat issue. Can we add a test in an appropriate place that both the old and new flags are true for an enterprise auth server?

Reply:
i placed the test in `auth_with_roles`, because we convert using the proto object there: https://github.com/gravitational/teleport/pull/34792/commits/5f11b1239f2036257ebaa4c10125572870dc6446